README_zh.md

```diff
@@ -24,6 +24,56 @@ Golang 1.16+
 go get -u github.com/zc2638/swag@v1.4.3
 ```
 
+## 使用注意：
+time.time类型使用jsonTag来将其解析为string类型
+```go
+type XXX struct{
+  Now time.Time `json:",string"`
+}
+```
+
+不兼容如下解析：
+```go
+type XX struct {
+   Attributes1 []map[string]string //建议把map替换成struct
+   Attributes2 [][]string
+   Attributes3 map[string][][]string
+   Attributes4 map[string][]map[string]string //建议把map替换成struct
+   Attributes5 map[string]XX //不支持对结构体嵌套自身解析
+}
+```
+
+**对象嵌套自身处理方式：**
+
+生成swagger文档后在需手动为嵌套字段添加example，map元素嵌套结构体自身示例如下：
+```
+type RuleOption struct {
+MatchSrc   string                 `json:"match_src,omitempty"`
+SubOptions map[string]*RuleOption `json:"sub_options,omitempty"`
+}
+## 修改swagger文档中，为sub_options添加example
+api.RuleOption:
+    type: object
+    properties:
+      match_src:
+        type: string
+      name:
+        type: string
+      ops:
+        type: array
+        items:
+          type: string
+      sub_options:
+        type: object
+        additionalProperties:
+          $ref: '#/definitions/api.RuleOption'
+          example: {"match_src":"string","name":"string","ops":"string","sub_options":{},"values":["string"]}
```

Comment:
I tested at `editor.swagger.io` and I think this `example` might not work as expected.

<img width="1436" alt="image" src="https://github.com/zc2638/swag/assets/28284116/b8526388-bf27-4342-a489-e7d80a92ffb0">

Reply:
Yes, might not a good example. Then follow the swagger official default display is fine